File-Formats/PDF/Convert-HTML-To-PDF/docker.md

```diff
@@ -7,91 +7,89 @@ documentation: UG
 keywords: Assemblies
 ---
 
+# Convert HTML to PDF file in Linux docker container
+
 The Syncfusion HTML to PDF converter is a .NET library that converts HTML or web pages to PDF.  This library converts an HTML to PDF in a Linux [docker](https://www.docker.com/why-docker/) container.
 
-# Steps to convert HTML to PDF in Linux docker container:
+## Steps to convert HTML to PDF in Linux docker container
 
-Create a new ASP.NET Core application and enable the docker support with Linux as a target OS.
+1. Create a new ASP.NET Core application and enable the docker support with Linux as a target OS.
 ![Convert HTMLToPDF Docker Step1](htmlconversion_images/DockerStep1.png)
 ![Convert HTMLToPDF Docker Step2](htmlconversion_images/DockerStep2.png)
 
-Install the [Syncfusion.HtmlToPdfConverter.Net.Linux](https://www.nuget.org/packages/Syncfusion.HtmlToPdfConverter.Net.Linux/) NuGet package as a reference to your .NET Core application [NuGet.org](https://www.nuget.org/)
+2. Install the [Syncfusion.HtmlToPdfConverter.Net.Linux](https://www.nuget.org/packages/Syncfusion.HtmlToPdfConverter.Net.Linux/) NuGet package as a reference to your .NET Core application [NuGet.org](https://www.nuget.org/).
 ![Convert HTMLToPDF Docker Step3](htmlconversion_images/DockerStep3.png)
 
-Include the following commands in the docker file to install the dependent packages in the docker container.
+3. Include the following commands in the docker file to install the dependent packages in the docker container.
 
-{% highlight c# tabtitle="C#" %}
+   {% highlight c# tabtitle="C#" %}
 
-RUN apt-get update && \
-     apt-get install -yq --no-install-recommends \ 
-     libasound2 libatk1.0-0 libc6 libcairo2 libcups2 libdbus-1-3 \ 
-     libexpat1 libfontconfig1 libgcc1 libgconf-2-4 libgdk-pixbuf2.0-0 libglib2.0-0 libgtk-3-0 libnspr4 \ 
-     libpango-1.0-0 libpangocairo-1.0-0 libstdc++6 libx11-6 libx11-xcb1 libxcb1 \ 
-     libxcursor1 libxdamage1 libxext6 libxfixes3 libxi6 libxrandr2 libxrender1 libxss1 libxtst6 \ 
-     libnss3 libgbm1
+    RUN apt-get update && \
+        apt-get install -yq --no-install-recommends \ 
+        libasound2 libatk1.0-0 libc6 libcairo2 libcups2 libdbus-1-3 \ 
+        libexpat1 libfontconfig1 libgcc1 libgconf-2-4 libgdk-pixbuf2.0-0 libglib2.0-0 libgtk-3-0 libnspr4 \ 
+        libpango-1.0-0 libpangocairo-1.0-0 libstdc++6 libx11-6 libx11-xcb1 libxcb1 \ 
+        libxcursor1 libxdamage1 libxext6 libxfixes3 libxi6 libxrandr2 libxrender1 libxss1 libxtst6 \ 
+        libnss3 libgbm1
 
-{% endhighlight %}
+   {% endhighlight %}
 
-![Convert HTMLToPDF Docker Step4](htmlconversion_images/DockerStep4.png)
+   ![Convert HTMLToPDF Docker Step4](htmlconversion_images/DockerStep4.png)
 
-Add an Export to the PDF button in the index.cshtml.
+4. Add an Export to the PDF button in the index.cshtml.
```

Comment:
Add a new button in the index.cshtml as shown below

Reply:
content modified